### PR TITLE
Add querystring to Cleanup Cluster Nodes tabs

### PR DESCRIPTION
**Pull request by dereknola**
## Description
Allows pointing directly to the Tabs for K3s/RKE1/RKE2 when referencing this page.
image

### CODE REVIEW

**Comment by martyav** [comment]
Docusaurus docs on this feature: https://docusaurus.io/docs/markdown-features/tabs#query-string

This probably also applies to our other pages containing tabs

**Review by martyav — COMMENTED** [review]
The modified lines look well-formed, I'm just wondering if we should add queryStrings more generally, and if that should be handled in another PR.

File: docs/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 
 ## Cleaning up Nodes
 
-<Tabs>
+<Tabs queryString="distro">
```
Comment (btat) [inline docs/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md:125]:
```suggestion
<Tabs groupId="k8s-distro" queryString>
```

File: versioned_docs/version-2.7/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 
 ## Cleaning up Nodes
 
-<Tabs>
+<Tabs queryString="distro">
```
Comment (btat) [inline versioned_docs/version-2.7/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md:125]:
```suggestion
<Tabs groupId="k8s-distro" queryString>
```

File: versioned_docs/version-2.8/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md [hunk 3]
```diff
@@ -122,7 +122,7 @@ Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 
 ## Cleaning up Nodes
 
-<Tabs>
+<Tabs queryString="distro">
```
Comment (btat) [inline versioned_docs/version-2.8/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md:125]:
```suggestion
<Tabs groupId="k8s-distro" queryString>
```

File: versioned_docs/version-2.9/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md [hunk 4]
```diff
@@ -122,7 +122,7 @@ Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 
 ## Cleaning up Nodes
 
-<Tabs>
+<Tabs queryString="distro">
```
Comment (btat) [inline versioned_docs/version-2.9/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md:125]:
```suggestion
<Tabs groupId="k8s-distro" queryString>
```

**Review by btat — CHANGES_REQUESTED** [review]
Suggested an alternate attribute and value combo since we already use `groupId` for tab syncing and have an existing value for the RKE1/RKE2/K3s grouping.

**Pull request merged**